integration/daemon/daemon_test.go

```diff
@@ -146,3 +154,138 @@ func TestConfigDaemonSeccompProfiles(t *testing.T) {
 		})
 	}
 }
+
+func TestDaemonProxy(t *testing.T) {
+	skip.If(t, runtime.GOOS == "windows", "cannot start multiple daemons on windows")
```

Comment:
If someone knows if  we can somehow test this in Windows CI, let me know 😅

Reply:
Looks like we may have to skip this on rootless as well; it's not connecting to the "proxy". Not exactly sure why it would not be supported by rootless though, but perhaps something in how networking is set up there? 🤔 

```
=== RUN   TestDaemonProxy/environment_variables
    daemon_test.go:183: assertion failed:  (received string) != example.org:5000 (string)
    --- FAIL: TestDaemonProxy/environment_variables (0.53s)
```